assign_rights/test_helpers.py

```diff
@@ -60,17 +60,18 @@ def set_dates():
 def add_rights_shells(count=15):
     for x in range(count):
         start_date, end_date, start_date_period, end_date_period, end_date_open = set_dates()
-        new_shell = RightsShell.objects.create(determination_date=random_date(10, 0), note=random_string(), start_date=start_date, end_date=end_date, start_date_period=start_date_period, end_date_period=end_date_period, end_date_open=end_date_open)
+        new_shell = RightsShell(determination_date=random_date(10, 0), note=random_string(), start_date=start_date, end_date=end_date, start_date_period=start_date_period, end_date_period=end_date_period, end_date_open=end_date_open)
         basis = random.choice(["copyright", "policy", "donor", "statute", "license"])
         setattr(new_shell, "rights_basis", basis)
         if basis in ["copyright", "statute"]:
-            setattr(new_shell, "jurisdiction", "us")
+            new_shell.jurisdiction = "us"
         if basis == "copyright":
-            setattr(new_shell, "copyright_status", random.choice(["copyrighted", "public domain", "unknown"]))
+            new_shell.copyright_status = random.choice(["copyrighted", "public domain", "unknown"])
         if basis == "statute":
-            setattr(new_shell, "statute_citation", random_string())
+            new_shell.statute_citation = random_string()
         if basis == "license":
-            setattr(new_shell, "license_terms", random_string())
+            new_shell.license_terms = random_string()
```

Comment:
does it make sense to make this `elif`s since they're mutually exclusive from line 68?